pilot/lib/pilot/tester_manager.rb

```diff
@@ -27,6 +27,32 @@ def add_tester(options)
       UI.success("Successfully added tester #{config[:email]} to app #{app.name} in group(s) #{group_names}")
     end
 
+    def add_tester_to_build(options)
+      start(options)
+      app = find_app(apple_id: config[:apple_id], app_identifier: config[:app_identifier])
+      UI.user_error!("You must provide either a Apple ID for the app (with the `:apple_id` option) or app identifier (with the `:app_identifier` option)") unless app
+
+      email_param = config[:email]
+      UI.user_error!("You must provide 1 or more emails (with the `:email` option)") unless email_param
+
+      if config[:build_number].nil?
+        build = app.get_builds[0]
+      else
+        builds = app.get_builds.select do |b|
+          config[:build_number] == b.version
+        end
+        UI.user_error!("Build #{config[:build_number]} not found.") unless builds.count == 1
+        build = builds[0]
+      end
+      UI.success("Build: #{build.version} id: #{build.id}")
```

Comment:
I think this doesn't need to be displayed prominently as a success message. Maybe `UI.message` or `UI.verbose` would be more appropriate, what do you think? 😊

Reply:
Yes, that's better, I didn't know these functions yet. Maybe like this?

```suggestion
      UI.verbose("Build: #{build.version} id: #{build.id}")
```

Reply:
Yup, sounds good!